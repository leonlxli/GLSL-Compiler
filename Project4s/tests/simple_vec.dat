funct: main
gin: v, vec2, 1.0, 2.0
