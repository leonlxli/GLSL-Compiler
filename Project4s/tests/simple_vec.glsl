vec2 v;
float main() {
	v.x = 0.0;
	return v.x;
}