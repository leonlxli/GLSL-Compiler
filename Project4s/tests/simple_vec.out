Result: 0.000000e+00
